### DISCUSSION TITLE
Ingestion is failing after replacing an expired SSL cert.

### DISCUSSION THREAD

**Discussion by mike-credisense** [datalust/seq-tickets#2250] [Help]
We are getting a "SSL Connection could not be established, an existing connection was forcibly closed" from Serlilog  after updating  an expiring SSL Certificate.

The certificates was installed and rebound to the urls
The front end works but not the ingestion port

**Comment by KodrAus** [comment]
Hi @mike-credisense :wave: Is this all resolved now?

You might find some details by checking Seq's ingestion log under _Settings > Diagnostics > Ingestion Log_. Otherwise if you send us a diagnostic report from that same screen we'll take a look.

**Comment by mike-credisense** [comment]
Yes all resolved now

Mike Wemyss

Security and Technical Analyst
Credisense Limited
+64 (0) 212579516 | ***@***.******@***.***> | credisense.co.nz<http://credisense.co.nz/>

ISO/IEC 27001:2013 | ISO/IEC 9001:2015 | Certificate No.NZP1107CA
